src/snyk/common/vscode/types.ts

```diff
@@ -27,5 +28,6 @@ export type DecorationOptions = vscode.DecorationOptions;
 export type ThemeColor = vscode.ThemeColor;
 export type ThemableDecorationInstanceRenderOptions = vscode.ThemableDecorationInstanceRenderOptions;
 export type CodeActionProviderMetadata = vscode.CodeActionProviderMetadata;
+export type LineDecorations = DecorationOptions[];
```

Comment:
This is a wrong place to define custom types. This file is used to map vscode to internal vscode types.